src/modules/processes.nix

```diff
@@ -120,7 +120,7 @@ in
 
     procfile =
       pkgs.writeText "procfile" (lib.concatStringsSep "\n"
-        (lib.mapAttrsToList (name: process: "${name}: ${pkgs.writeShellScript name process.exec}")
+        (lib.mapAttrsToList (name: process: "${name}: exec ${pkgs.writeShellScript name process.exec}")
```

Comment:
I had this same idea, but does this not break this feature? https://github.com/cachix/devenv/pull/692

Reply:
Maybe we should also do `"exec ${process.exec}"`?

Reply:
It does not, as `writeShellScript` will generate a store path with the script